dags/longitudinal.py

```diff
@@ -77,7 +77,7 @@
                       email=["telemetry-alerts@mozilla.com", "chudson@mozilla.com"],
                       execution_timeout=timedelta(hours=10),
                       release_label="emr-5.0.0",
-                      instance_count=5,
+                      instance_count=30,
```

Comment:
Using many instances might cause to hit some AWS limits when multiple jobs are scheduled at the same time; let's try to not do that unless there is a good reason for it.